smieci/react/colorPicker/src/Register.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react'
+
+type Props = {}
+
+function Register({ }: Props) {
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
+
+    const handleRegister = async (e: React.FormEvent) => {
+        e.preventDefault();
+        const response = await fetch('http://localhost:8080/api/v1/auth/register', {
+            method: 'POST',
+            headers: {
+                'Content-Type': 'application/json'
+            },
+            body: JSON.stringify({ email, password })
+        });
+
+        if (response.ok) {
+            alert('register success');
+            window.location.href = '/login';
+        } else {
+            console.log(response);
```

Comment:
1) install prettier plugin . instull eslint plugin in intelij

2) get into intelij command + , 

![Screenshot 2024-10-28 at 20 10 20](https://github.com/user-attachments/assets/acc4c466-37ec-41ff-9356-e56c903f8d19)

here Run on save 

then go  ALL ACTIONS ON SAVE

![Screenshot 2024-10-28 at 20 10 46](https://github.com/user-attachments/assets/f1d5ddb8-400f-4ae6-b0a0-674edd0e2429)


check clean up boxes. apply ok. then introduce some changes into file and save it. it should be reformatted. 
Prettier - is a tool for pretty javascript (ts) code 
Eslint - is a tool for safe javascript (ts) code  


make sure your eslint is ON too.

 
![Screenshot 2024-10-28 at 20 12 08](https://github.com/user-attachments/assets/3c8fb33b-f6f7-4383-bdff-2b0040ce1b10)